ocs_ci/templates/workloads/cephfs_stress/cephfs_filesystem_monitor.py

```
@@ -0,0 +1,633 @@
+
+def signal_handler(signum, frame):
+    """Handle shutdown signals gracefully"""
+    logger.info(f"Received signal {signum}, initiating graceful shutdown...")
+    shutdown_event.set()
+
+
+signal.signal(signal.SIGTERM, signal_handler)
+signal.signal(signal.SIGINT, signal_handler)
+
+
+def run_command_with_timeout(command: str, timeout: int, shell: bool = True) -> Tuple[bool, str, float]:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)